frontend/pages/projects/new.tsx

```diff
@@ -59,28 +80,32 @@ const Project: PageComponent<ProjectProps, FormPageLayoutProps> = () => {
   });
 
   const handleCreate = useCallback(
-    (data: ProjectForm) =>
-      // TO DO!
-      // createProject.mutate(data, {
-      //   onError: handleServiceErrors,
-      //   onSuccess: () => {
-      //     push('/project-developers/pending');
-      //   },
-      // }),
-      console.log(data),
-    []
+    (data: ProjectCreationPayload) =>
+      createProject.mutate(data, {
+        onError: (error) => {
+          const { errorPages, fieldErrors } = getServiceErrors<ProjectForm>(error, formPageInputs);
+          console.log(error, errorPages, fieldErrors);
+          fieldErrors.forEach((field) => setError(field, { message: '' }));
```

Comment:
What's the purpose of this line? Isn't this displaying empty error messages?

Reply:
Yes, it is not finished. I will remove it for now